templates/java/LocalPubSub/src/main/java/com/localpubsub/App.java

```diff
@@ -0,0 +1,36 @@
+package com.localpubsub;
+
+import software.amazon.awssdk.aws.greengrass.GreengrassCoreIPCClientV2;
+import software.amazon.awssdk.aws.greengrass.SubscribeToTopicResponseHandler;
+import software.amazon.awssdk.aws.greengrass.model.SubscribeToTopicResponse;
+import software.amazon.awssdk.aws.greengrass.model.UnauthorizedError;
+
+import java.util.Arrays;
+
+public class App {
+
+    public static void main(String[] args) {
+        String topic = args[0];
+        String message = String.join(" ", Arrays.copyOfRange(args, 1, args.length));
+        try (GreengrassCoreIPCClientV2 ipcClient = GreengrassCoreIPCClientV2.builder().build()) {

```

Comment:
while technically correct, I'm not sure if this sets our customers up for success. I'd recommend putting the client as a field of the class and initializing it when starting up. Do not close it.